ci/functional/test_main.sh

```diff
@@ -104,3 +116,4 @@ for node in ${tnodes//,/ }; do
         mv "$old_name" "$new_name"
     fi
 done
+"$hardware_ok"
```

Comment:
Wow, I didn't know bash supported this syntax for the final script return code!

Reply:
Bash functions exit with the exit value of their final statement.  Since `true` and `false` are executables in Linux returning `0` and `1` respectively, they can be used as the final statement of a function.

Reply:
Thanks. I've been in the habit of explicitly returning the $? or PIPESTATUS as necessary.